### PR TITLE
Update Kafka version to 2.2

### PR DESCRIPTION
**Pull request by noslowerdna**
Upgrading the Kafka client version to 2.2.0

fixes https://github.com/cerner/common-kafka/issues/14

### CODE REVIEW

File: common-kafka-admin/src/main/java/com/cerner/common/kafka/admin/KafkaAdminClient.java [hunk 1]
```diff
@@ -715,7 +719,12 @@ public void addTopicPartitions(String topic, int partitions) {
             throw new IllegalArgumentException("consumerGroup cannot be null, empty or blank");
 
         try {
-            return getAdminClient().describeConsumerGroup(consumerGroup);
+            AdminClient.ConsumerGroupSummary summary = getAdminClient().describeConsumerGroup(consumerGroup, operationTimeout);
+            if (summary.state().equals(ConsumerGroupState.DEAD.toString())) {
+                throw new AdminOperationException("Consumer group not found: " + consumerGroup);
```
Comment (bbaugher) [inline common-kafka-admin/src/main/java/com/cerner/common/kafka/admin/KafkaAdminClient.java]: Add doc about this case

**Comment by bbaugher** [comment]
We can delete our ByteArrayConverter since it Kafka has its version [here](https://issues.apache.org/jira/browse/KAFKA-4783). Update the README too

**Comment by noslowerdna** [comment]
> We can delete our ByteArrayConverter

done: https://github.com/cerner/common-kafka/commit/fb7585eff0add3e1aaca500dfbea91c65ad3fe58

File: common-kafka-admin/src/main/java/com/cerner/common/kafka/admin/KafkaAdminClient.java (hunk 1 above)
Comment (noslowerdna) [reply common-kafka-admin/src/main/java/com/cerner/common/kafka/admin/KafkaAdminClient.java]: It's been a couple weeks but I think I added this to preserve the original behavior... agree it is probably better to return the DEAD group summary and just fix the tests and whatever code is calling this to check for dead

Comment (noslowerdna) [reply common-kafka-admin/src/main/java/com/cerner/common/kafka/admin/KafkaAdminClient.java]: Done (both) - https://github.com/cerner/common-kafka/commit/8d357d9b12a0fca17910b0bb3728fb89030cfd2e

**Pull request merged**